src/weapon.c

```diff
@@ -131,11 +131,12 @@ void weapon_spread(struct Player* p, float* d) {
 		case WEAPON_SMG: spread = 0.012F; break;
 		case WEAPON_SHOTGUN: spread = 0.024F; break;
 	}
-	d[0] += (ms_rand() - ms_rand()) / 16383.0F * spread * (p->input.buttons.rmb ? 0.5F : 1.0F)
+
+	d[0] += (ms_rand() - ms_rand()) / 16383.0F * spread * (p->is_ads ? 0.5F : 1.0F)
 		* ((p->input.keys.crouch && p->weapon != WEAPON_SHOTGUN) ? 0.5F : 1.0F);
-	d[1] += (ms_rand() - ms_rand()) / 16383.0F * spread * (p->input.buttons.rmb ? 0.5F : 1.0F)
+	d[1] += (ms_rand() - ms_rand()) / 16383.0F * spread * (p->is_ads ? 0.5F : 1.0F)
```

Comment:
This is an issue for the recoil of other players, since `is_ads` is never set for them (weapon_spread is also used for their visible tracers). I decided to close this PR in favor over maybe simpler changes I made to solve the whole problem.

Reply:
ACK

Reply:
:(